src/xenia/kernel/xboxkrnl/xboxkrnl_modules.cc

```diff
@@ -166,7 +166,13 @@ dword_result_t XexGetProcedureAddress(lpvoid_t hmodule, dword_t ordinal,
       *out_function_ptr = ptr;
       result = X_STATUS_SUCCESS;
     } else {
-      XELOGW("ERROR: XexGetProcedureAddress ordinal not found!");
+      if (is_string_name) {
+        XELOGW("ERROR: XexGetProcedureAddress export '{}' in '{}' not found!",
+               string_name, module->name());
+      } else {
+        XELOGW("ERROR: XexGetProcedureAddress ordinal {:X} in '{}' not found!",
```

Comment:
This isn't clear if the ordinal doesn't have any A-F characters in it — if we need it as hexadecimal, possibly it should be prefixed with 0x.

Reply:
Fair enough, I'll just log in both dec and hex since it's more or less Xenia convention to refer to ordinals in hex (ie export tables, etc).